lua/bufferline/diagnostics.lua

```diff
@@ -73,7 +73,18 @@ end
 local get_diagnostics = {
   nvim_lsp = function()
     if is_nightly then
-      return vim.diagnostic.get()
+      local results = {}
+      diagnostics = vim.diagnostic.get()
```

Comment:
The same applies here as well actually, I only just saw this one
```suggestion
      local diagnostics = vim.diagnostic.get()
```